integration-tests/build.gradle

```diff
@@ -34,6 +34,8 @@ test {
     jacoco {
     }
     environment "AUDIT_SIGNING_KEY_ALIAS", "alias/local-audit-payload-signing-key-alias"
+    environment "AUTHORIZE_REQUEST_LAMBDA_ARN", "arn:authorize-request:eu-west-2:6546546465"
```

Comment:
See comment about overriding config service, it might be better than hard-coding values in here.

Reply:
Updated